src/modules/main/sidebar/Sidebar.tsx

```diff
@@ -37,13 +40,20 @@ export const Sidebar = ({ basePath }: SidebarProps) => {
 
   const activeLinkIdx = getActiveLinkIdx(pathname.replace(basePath, ''), sidebarLinks);
 
+  const [open, setOpen] = useState(false);
+
+  const toggleOpen = () => setOpen(!open);
```

Comment:
setOpen(prevOpen => !prevOpen)
also add useCallback with [] deps array

Reply:
The solution has been applied